serverless.js

```diff
@@ -39,12 +39,9 @@ module.exports = {
     memorySize: 512,
     region: AWS_REGION,
     stage: SLS_STAGE,
-    httpApi: {
-      payload: '2.0',
-      cors: {
```

Comment:
Don't we need CORS for these new endpoints?

Reply:
yeah actually im not sure why and how its still working! ill look into it

Reply:
so actually, i think its because of:
https://github.com/yldio/asap-hub/blob/19514f4c94a620c2ca64755ef7d572cd71628a69/apps/asap-server/src/app.ts#L94
we seem to be getting the following response:
![image](https://user-images.githubusercontent.com/11645956/113559570-8c68b600-95f9-11eb-8ac3-d92b0d0be91f.png)

Previously with cors enabled in serverless we had:
![image](https://user-images.githubusercontent.com/11645956/113559596-95598780-95f9-11eb-9237-19bdad11a410.png)

I don't think it makes any difference, at least looking at the the asap hub (no sign of errors). The only thing I could think of is that the webhooks probably do not return the cors headers, but then I don't really think they need to anyway.